city_scrapers/spiders/il_board_of_examiners.py

```diff
@@ -0,0 +1,104 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlBoardOfExaminersSpider(CityScrapersSpider):
+    name = "il_board_of_examiners"
+    agency = "Illinois Board of Examiners"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.ilboe.org"]
+    start_urls = ["https://www.ilboe.org/board-information/board-meetings/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css(".minutes"):
+            meeting = Meeting(
+                title="Illinois Board of Examiners",
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        date_str = re.sub(r"(?<=\d)[a-z]+", "", item.css('.minuteDate::text').get())
+        time_str = item.css('.minuteTime::text').get()
+        datetimeObj = ''
+        datetimeObj = datetime.strptime(date_str + ':' + time_str, "%B %d, %Y:%I:%M %p")
+        return datetimeObj
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
```

Comment:
It doesn't look like this is working consistently (you can see some of the output in the [last test run](https://travis-ci.org/City-Bureau/city-scrapers/jobs/586594978)). One way to parse this out would be to first check if there are numbers anywhere in the location string and start the address there for cases like:
Illinois CPA Society
550 W. Jackson Blvd.
9th Floor Building Meeting Room
Chicago, Illinois

If it starts with a number then the whole thing is the address. For addresses that don't include any numbers like the one below you can default to using the last line as the address if there is more than one line.

Northern Illinois University
DeKalb, IL